diff --git a/base16-synthwave-84.vim b/base16-synthwave-84.vim
index 7b1258d..725d030 100644
--- a/base16-synthwave-84.vim
+++ b/base16-synthwave-84.vim
@@ -251,7 +251,7 @@ call <sid>hi("Define",       s:gui0E, "", s:cterm0E, "", "none", "")
 call <sid>hi("Delimiter",    s:gui0F, "", s:cterm0F, "", "", "")
 call <sid>hi("Float",        s:gui09, "", s:cterm09, "", "", "")
 call <sid>hi("Function",     s:gui0D, "", s:cterm0D, "", "", "")
-call <sid>hi("Identifier",   s:gui03, "", s:cterm03, "", "none", "")
+call <sid>hi("Identifier",   s:gui03, "", s:cterm07, "", "none", "")
 call <sid>hi("@variable",     s:gui08, "", s:cterm08, "", "none", "")
 call <sid>hi("Include",      s:gui0D, "", s:cterm0D, "", "", "")
 call <sid>hi("Keyword",      s:gui0E, "", s:cterm0E, "", "", "")
